README.md

```diff
@@ -28,7 +37,12 @@ c = c + d;
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
 
-<!-- Replace this comment with your answer -->
+The value of c after the code executes is 7.
+
+In the last line, the operator "=" is an assignment operator. In this case, the "=" operator assigns to c the result of what the expression to the right of the "=" operator evaluates to.
+
+the "+" operator (I believe) is the addition operator rather than the numeric conversion (unary?) operator because we are evaluating two variables that have been assigned number values. If the variable c was initialized with a string value, I think that '+' would be considered a numeric conversion operator. I'm a bit shaky on this concept---numeric conversions is a new concept!
+
```

Comment:
Very good! Yes, the behavior of `+` is  dependent on the kinds of inputs it's given. 
3 + 3 => 6   (add)
'3' + '3' => '33' (concatenate)
'3' + 3 => '33' (convert and concatenate)
3 + '3' => '33' (convert and concatenate)
 
For some bizarre, extreme examples, check this out:
https://www.destroyallsoftware.com/talks/wat